webapp/src/shared/components/Row/Row.css

```diff
@@ -0,0 +1,49 @@
+.row {
+  width: 80%;
+  height: 82px;
+  border-radius: 8px;
+  color: var(--color-light);
+  background-color: var(--color-dark);
+  border: 0;
+  position: relative;
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  flex-direction: row;
+  min-width: 230px;
+}
+
+.row_wrapper {
```

Comment:
suggestion: Consider removing the `row-wrapper` div, and use the flexbox properties and padding, something like this:

```
.row {
  width: 100%;
  min-width: fit-content;
  padding: 16px 28px;
  border-radius: 8px;
  color: var(--color-light);
  background-color: var(--color-dark);
  border: 0;
  cursor: pointer;
  display: flex;
  align-items: center;
  column-gap: 20px;
}

.row_text_wrapper > * {
  margin: 0;
  text-align: left;
  white-space: no-wrap;
}

.row-icon {
  margin-left: auto;
}
```

The avatar and the text are aligned by flexbox and `margin-left: auto;` pushes the icon to the [right](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container#using_auto_margins_for_main_axis_alignment).

If we add the `className` prop to the `Text` component, we could also use clases instead of the "exotic" `.row_text_wrapper > *` selector:
```
.row_text_title,
.row_text_subtitle {
  margin: 0;
  text-align: left;
  white-space: no-wrap;
}
```

Reply:
Agree with this

Reply:
I'm trying this, literally, and it doesn't work. I think we still need, at least, `row_text_wrapper`, in order to have `flex-direction: column;` maybe ?

Reply:
Should be working now. Any more comments on this topic?

Reply:
Thank you! Looks good to me.
We could remove the `height: 82px;` from the `.row'` class to allow for flexible height when there is only a title or subtitle.